hadoop-hdds/docs/content/shell/VolumeCommands.md

```diff
@@ -42,7 +42,7 @@ assign it to a user.
 |  Uri                           | The name of the volume.                                        |
 
 {{< highlight bash >}}
-ozone sh volume create --quota=1TB --user=bilbo /hive
+ozone sh volume create -ssq=1TB --user=bilbo /hive
```

Comment:
In design doc, it also provide a full name called ssQuota,  Not sure if there are other better names, or we can use 'spaceQuota' instead, but I think this can be explained in document.

Reply:
in the previous design, quota just describes the Space Quota, if we want to describe the Count Quota, which one is better to use ? quota is a little ambiguous to describe which one it wants to express. 
IMO, I prefer to use spaceQuota and countQuota instead

Reply:
Here we can refer to the naming rules of the [HDFS command](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-hdfs/HdfsQuotaAdminGuide.html), in HDFS:
The quota is a hard limit on the number of file.
The space quota is a hard limit on the number of bytes used by files.
So spaceQuota can be used here.